osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -196,5 +200,56 @@ public bool Failing
                 }
             };
         }
+
+        private class ActionableInfoWithNumberBox : ActionableInfo
+        {
+            public new Action<int> Action;
+
+            private OsuNumberBox numberBox;
+
+            public int NumberValue
+            {
+                get
+                {
+                    int.TryParse(numberBox.Text, out var val);
```

Comment:
Could just use `int.Parse` directly. `OsuNumberBox` should ensure there are no invalid chars here.

Reply:
not if string is empty or if string contains a number too big to fit in an int, altho in that case there should probably some error displayed to the user, but no idea how to do that.

iirc I do check somewhere else for empty string and then just do nothing lol

Reply:
Fair points. I'd probably leave this as it is, the tournament client isn't supposed to be the most user-friendly and polished thing at this point.